### ISSUE TITLE
Build error with parallel make

### ISSUE THREAD

**Issue by realnc** [chrender/fizmo#2]
There is an issue with parallel builds. It only happens on a clean source directory, directly after unpacking the tarball:

```
tar xf fizmo-0.8.0.tar.gz
cd fizmo-0.8.0
./configure
make -j4
...
pixel_interface.c:66:27: fatal error: drilbo/drilbo.h: No such file or directory
```

Note: This is only reproducible on the first unpack on the tarball. "make clean" or "make distclean" and then calling "./configure" again does not reproduce the issue.

This results in automated builds (Gentoo build system) failing to install the package.

This is on GCC 5.4.0 and GNU Make 4.2.1.

**Comment by wlhlm** [comment]
I can confirm this on Archlinux. Running parallel make immediately after extracting fails, but running the same make command again afterwards "magically" works without a problem.

Using `make -j1` fixes it all together.

- gcc 6.1.1
- make 4.2.1
- libsndfile 1.0.27

**Comment by chrender** [comment]
Fixed in fizmo-0.8.1. Thanks for reporting this.
